components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -291,7 +291,19 @@ initialize_fields (const util::TimeStamp& t0)
     f.get_header().get_tracking().update_time_stamp(t0);
   }
 
-  if (ifield>0) {
+  // Check the status of lat and lon values on reference grid:
+  // This option allows the user to set lat or lon in their own
+  // test or run setup code rather than by file.
+  bool skip_init_lat = true;
+  bool skip_init_lon = true;
+  if (ic_pl.isParameter("skip_init_lat")) {
+    bool skip_init_lat     = ic_pl.get<bool>("skip_init_lat");
```

Comment:
@AaronDonahue This is likely not what you want. You are not changing the value of the previous variable, but you are creating one with the same name, so `skip_init_lat` will always be true.

Your compiler should have issued a warning here, something like `Declaration shadows a local variable`.